code/game/objects/structures/crates_lockers/closets/secure/cargo_lockers.dm

```diff
@@ -21,8 +21,15 @@
 	open_door_sprite = "mining_door"
 
 /obj/structure/closet/secure_closet/quartermaster/populate_contents()
-	new /obj/item/radio/headset/headset_cargo(src)
+	new /obj/item/radio/headset/heads/qm(src)
 	new /obj/item/door_remote/quartermaster(src)
 	new /obj/item/organ/internal/eyes/cybernetic/meson(src)
 	new /obj/item/storage/bag/garment/quartermaster(src)
 	new /obj/item/clothing/accessory/medal/supply(src)
+	new /obj/item/fulton_core(src)
+	new /obj/item/extraction_pack(src)
+	new /obj/item/rcs(src)
+	new /obj/item/gun/energy/plasmacutter(src)
```

Comment:
They are supposed to go down on lavaland now?

Reply:
The intention is for them to be able to assist more easily in recovery of dead miners, if they wish. Half of their job revolves around coordinating and ensuring mining is able to function. This just gives them some additional tools for getting to, and recovering deceased minerss.

Reply:
After some discussion with Henri in coding chat, plasma cutter has been removed, with a GPS and crew monitor handed to the QM locker. The QM should be coordinating search and rescues FROM the mining base, not running into the wastes himself.